server/ecore-glsp/src/main/java/com/eclipsesource/glsp/ecore/EcoreModelIndex.java

```diff
@@ -150,4 +155,8 @@ public void remove(EObject eObject) {
 		}
 		semanticIndex.inverse().remove(eObject);
 	}
+	
+	public Set<Integer> getBidirectionalReferences() {
+		return bidirectionalReferences;
```

Comment:
HashCodes shouldn't be used as IDs, as they're not guaranteed to be unique. Even if hashcode are used as a temporary solution, I would use a String-type Set, and a helper method to get a String-ID from an EReference (And there, you may use Integer.toString(ref.hashCode()), so it can easily be replaced later on with proper stable IDs)

Reply:
@CamilleLetavernier The code has been adapted now - though it seems, that Github did not catch up with the squashed branches.

Is this now as it should be?